drivers/ieee802154/ieee802154_cc13xx_cc26xx_subg.c

```diff
@@ -66,6 +68,12 @@ static uint32_t overrides_sub_ghz[] = {
 	(uint32_t)0xFFFFFFFF
 };
 
+/** RF patches to use (note: RF core keeps a pointer to this, so no stack). */
```

Comment:
Was it necessary to move this? It would be best to reduce the PR to only the relevant code changes.

Reply:
Hi, yes it is. Let me explain:
`RF_open()` from TI's library expects the `RF_Mode* pRfMode` parameter with the operation mode & patches to apply to the radio module. Its implementation stores this pointer to an internal structure for lazy usage (the radio will be initialized on first use):
```c
/* Register the mode to the client */
pObj->clientConfig.pRfMode = pRfMode;
````

Actually, the radio is [re]initialized during each power up sequence, including every time after suspend. 
So the catch is... the structure was initialized on stack, which would get replaced once the `ieee802154_cc13xx_cc26xx_subg_init` function returned, so the pointer stored inside RF lib's `clientConfig` would become invalid. So, instead, I allocate it inside DATA section => problem solved.

Reply:
Yes - good catch! Thank you